src/lib/holocene/combobox/combobox.svelte

```diff
@@ -345,7 +355,7 @@
   }
 
   .combobox-wrapper {
-    @apply surface-primary flex h-10 w-full flex-row items-center rounded-lg border border-primary text-sm dark:focus-within:surface-primary focus-within:border-interactive focus-within:outline-none focus-within:ring-4 focus-within:ring-primary/70;
+    @apply surface-primary flex h-10 w-full flex-row items-center rounded-lg border-2 border-primary text-sm dark:focus-within:surface-primary focus-within:border-interactive focus-within:outline-none focus-within:ring-4 focus-within:ring-primary/70;
```

Comment:
Now that the border is 2px, the black makes it look extra chonky. Looks like the designs specify subtle, maybe we use that?
<img width="987" alt="Screenshot 2024-05-21 at 2 46 58 PM" src="https://github.com/temporalio/ui/assets/11775628/488ad7aa-9164-4e97-8c6f-ee97080ad215">

Reply:
🤌